src/info/info.nim

```diff
@@ -0,0 +1,72 @@
+import std/[algorithm, json, os, sets, terminal]
+import ".."/cli
+
+proc parseTrackConfig(trackDir: string): JsonNode =
+  let trackConfigPath = trackDir / "config.json"
+  if fileExists(trackConfigPath):
+    try:
+      result = parseFile(trackConfigPath)
+    except:
+      stderr.writeLine getCurrentExceptionMsg()
+      quit(1)
+  else:
+    stderr.writeLine "Missing file: " & trackConfigPath
+    quit(1)
+
+proc getConcepts(j: JsonNode): HashSet[string] =
+  ## Returns the slug of every concept.
+  for item in j["concepts"]:
+    result.incl item["slug"].getStr()
+
+proc getPrereqs(j: JsonNode): HashSet[string] =
+  ## Returns the dedeuplicated values of every practice exercise `prerequisites` key.
```

Comment:
```suggestion
  ## Returns the deduplicated values of every practice exercise `prerequisites` key.
```

Reply:
Fixed, along with the same typo elsewhere.